scripts/make-identifier-regexen.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const ID_Start = require('unicode-9.0.0/Binary_Property/ID_Start/code-points.js');
+const ID_Continue = require('unicode-9.0.0/Binary_Property/ID_Continue/code-points.js');
+const Other_ID_Start = require('unicode-9.0.0/Binary_Property/Other_ID_Start/code-points.js');
+
+
+function asFourHexDigits(cp) {
+  return cp.toString(16).padStart(4, 0);
```

Comment:
Implicit conversion of `0` to string. Gross.

Reply:
:| Had this right at some point...

Fixed.